src/cozmo/action.py

```diff
@@ -219,9 +210,21 @@ def _recv_msg_robot_completed_action(self, evt, *, msg):
         elif result == types.INTERRUPTED:
             self._set_failed('interrupted', 'Action was interrupted')
 
+        # Can add additional if statements for other ActionResults to exposure more specific failure reasons to users
```

Comment:
typo: exposure should be expose. We also try and wrap at about 80 chars for the SDK. I think we should just remove this comment and add some more detail in the else block though (about how we're summarizing the types by their category).